src/OrchardCore.Modules/OrchardCore.Setup/Views/Setup/Index.cshtml

```diff
@@ -173,7 +173,11 @@
                 </div>
 
                 <span asp-validation-for="ConnectionString" class="text-danger"></span>
-                <span class="text-muted form-text small">@T["The connection string to your database instance. e.g., Data Source=<em>localhost</em>;Initial Catalog=<em>Orchard</em>;User Id=<em>userid</em>;Password=<em>password</em>"]</span>
+                <span id="connectionStringHint" class="text-muted form-text small"
```

Comment:
I didn't realize this, but we should instead add a `SampleConnectionString` property in `DatabaseProvider` and use it in the loop. So we don't have to list the database in the view. You can reuse the `"The connection string to your database instance. e.g., "` text in the localization, and use a `{0}` placeholder to inject the sample connection string.

Then in the `AddDataAccess` method where the providers are registered, provide the sample connection string.

Sorry for the misunderstanding.

Reply:
> but we should instead add a SampleConnectionString property in DatabaseProvider

I don't think it's a better idea, while this only used in two views, but give me a minute to think if you unlike the current implementation

> You can reuse the "The connection string to your database instance. e.g., " text in the localization

Ya, we can do this